tests/core/fmf-id/test.sh

```diff
@@ -28,7 +28,8 @@ rlJournalStart
         rlAssertGrep "some-package" $rlRun_LOG
 
         rlRun -s "tmt test -vvvv show /test-with-invalid-ref" 2
-        rlAssertGrep "The 'ref' field must be a string, got 'int'." $rlRun_LOG
+        rlAssertGrep "warn: /test-with-invalid-ref:require .* is not valid under any of the given schemas" $rlRun_LOG
+        rlAssertGrep "Field 'ref' can be not set or string, 'int' found." $rlRun_LOG
```

Comment:
`can be not set or string`? well, I would rather go with

```
Field 'ref' must be unset or a string, 'int' found.
```

Would `must` be more correct here? Can  it be something else?

Reply:
Fixed in ede71004.